### ISSUE TITLE
Setting a non-runtime config item should produce a warning

### ISSUE THREAD

**Issue by darkstar** [dosbox-staging/dosbox-staging#1027]
**Describe the bug**
In the late 90's I did some toying around with various demoscene effect. One of these effects was a copper bar. I made several small executables that displayed one or more horizontal color bars moving in a smooth up/down motion.

I coded these on (I think) a 486 with a Trident VGA card, and I'm certain they worked on that machine on pure DOS.

However, they do not display anything at all on DosBox-staging.

**To Reproduce**
Run the attached executables in DosBox

**Expected behaviour**
some fancy colored bars should appear, and maybe move up or down the screen (depending on the exe file... I don't remember what effect each EXE file does, it's been over 20 years by now...)

**Environment (please complete the following information):**
- Windows 7
- x86_64

**Version of dosbox-staging:**
- dosbox-staging version 0.77.0-alpha-760-gdc029

**Configuration parameters:**
Basically I tried svga_se, vgaonly, svga_paradise and some others. I also tried the `vertical retrace poll debug line` parameter but it didn't show anything

**Additional context**
I'm not sure my code is (technically) correct... but it did work on real hardware so I'm not sure if this qualifies as a bug or not...

The programs are pretty small and straightforward. I will try and see if I can find the source code, but basically it's more or less waiting for bit 3 of 0x3da, writing to 0x3c8, and waiting for bit 1 of 0x3da.
[copper.zip](https://github.com/dosbox-staging/dosbox-staging/files/6528684/copper.zip)

**Comment by kcgen** [comment]
Try this `dosbox.conf`

``` ini
[sdl]
fullscreen = true
capture_mouse = nomouse middlerelease
output = texturepp

[dosbox]
machine = vgaonly
memsize = 2
startup_verbosity = quiet

[render]
aspect = true

[cpu]
core    = dynamic
cycles  = 7000

[sblaster]
sbtype = none

[midi]
mididevice = none

[gus]
gus = false


[speaker]
pcspeaker=false
tandy=off
disney=false

[dos]
xms = false
ems = false
umb = false

[autoexec]
@echo off
mount c . > NUL
c:
pause
copper
pause
copper2
pause
copper3
pause
copper5
pause
copper6
pause
copper7
pause
exit
```

I'm atleast seeing stuff with it :-)

**Comment by darkstar** [comment]
Indeed. It seems that one cannot switch the machine type to `vgaonly` while the machine is running (which is what I tried)

When I set the machine type in the config, it works.

I'm not sure if the machine-switching is a bug? I think there should at least be a warning if the switch at runtime is not supported...

**Comment by kcgen** [comment]
> I think there should at least be a warning if the switch at runtime is not supported...

Yes - I've updated this issue's title to reflect this. Thanks for the suggestion!

**Comment by Wengier** [comment]
@darkstar You mentioned the `vertical retrace poll debug line` parameter. Just let you know it is not a supported option of DOSBox Staging but of another fork (DOSBox-X). There a no such option in DOSBox-Staging.

**Comment by darkstar** [comment]
> @darkstar You mentioned the `vertical retrace poll debug line` parameter. Just let you know it is not a supported option of DOSBox Staging but of another fork (DOSBox-X). There a no such option in DOSBox-Staging.

lol okay, I definitely did check dosbox-staging's git repo while searching for possible config file parameter here on github, I must have somehow moved over to dosbox-x without noticing. I could have sworn I saw a reference config in this repo, but apparently not :-D 

This is a little bit embarassing ...

**Comment by Burrito78** [comment]
@kcgen How about a log message in Loguru? I think that is the minimal result when DOSBox goes "nope".

**Comment by FeralChild64** [comment]
The log is, IMHO, not enough - for example, it is not visible in fullscreen mode. zthe CONFIG.COM itself should print out a warning to the DOS console.

**Comment by johnnovak** [comment]
Agree with @FeralChild64, the config command should print an error in such cases.
